infra/aws/cloudformation/iam/cdn.packages.k8s.io/cloudformation.yaml

```diff
@@ -0,0 +1,210 @@
+Resources:
+  #########################################
+  # Terraform State                       #
+  #########################################
+
+  # TFStateBucket (packages-k8s-io-tfstate) is S3 bucket for storing Terraform
+  # state
+  TFStateBucket:
```

Comment:
I suggest enabling versioning on this bucket.

Reply:
+1